.github/workflows/pr.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       run: if [[ -n $(gofmt -l .) ]]; then echo "please run gofmt"; exit 1; fi
     - name: generated files should not be modified
       run: |
-        go generate ./...
+        go generate -tags=forcegen ./...
```

Comment:
I'm not sure we can use `-tags=forcegen` that easily. This imo defines the build tag for building the generator, but not how we run the commands to actually run later. At least when I'm running locally `go generate -tags=forcegen ./...` then it doesn't force the generation.

Reply:
Maybe we can get it working though - in the packages where we have directives
```
//go:generate go run main.go
```
we define a global constant which is defined when we call `go generate -tags=forcegen ./...` and then refer to it in the subgenerators?